docker-compose.yml

```diff
@@ -101,6 +101,47 @@ services:
       - 5432:5432
     volumes:
       - /var/lib/postgresql/data
+  rbac:
+    image: quay.io/cloudservices/rbac:latest
+    restart: on-failure
+    environment:
+      - REDIS_PORT=6379
+      - REDIS_HOST=redis
+      - DATABASE_HOST=db
+      - DATABASE_PORT=5432
+      - DATABASE_NAME=insights
+      - DATABASE_USER=insights
+      - DATABASE_PASSWORD=insights
+      - API_PATH_PREFIX=/api/rbac
+      - DEVELOPMENT=True
+    depends_on:
+      - build-rbac-db
+      - db
+      - redis
+    links:
+      - db
+      - redis
+    ports:
+      - 9002:8080
+    volumes:
+      - ./config/devel/rbac/permissions.json:/opt/app-root/src/rbac/management/role/permissions/compliance.json:z
+      - ./config/devel/rbac/definitions.json:/opt/app-root/src/rbac/management/role/definitions/compliance.json:z
+  build-rbac-db:
+    image: quay.io/cloudservices/rbac:latest
+    environment:
+      - TENANT_PARALLEL_MIGRATION_MAX_PROCESSES=4
+      - TENANT_PARALLEL_MIGRATION_CHUNKS=2
+      - ACCESS_CACHE_CONNECT_SIGNALS=False
+      - MAX_SEED_THREADS=2
```

Comment:
This won't ever work, you don't even have the ENV variables that allow the container to connect to the DB. Check the logs of the container...

Reply:
Thanks